client/src/router.ts

```diff
@@ -80,6 +81,13 @@ const router = createRouter({
   },
 });
 
+/// Remove layer query from scorecard page.
```

Comment:
is it difficult to prevent it from being added at all?

Reply:
It happens through the nav bar through the native router-links. From the searches I did, I couldn't find much about it. I also removed the ? From the scorecord path but that didn't do anything. If I don't do this, the about page is also http://localhost:8080/about?layer=lead-service-lines-by-water-system